tests/integration/test_vulnerability_detector/test_scan_results/test_alert_vulnerability_removal.py

```diff
@@ -1,181 +1,294 @@
-
 import pytest
 
-from wazuh_testing.modules import vulnerability_detector as vd
-from wazuh_testing.tools import ALERT_FILE_PATH, LOG_FILE_PATH
-from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools import configuration
+from wazuh_testing.modules.vulnerability_detector.utils import check_vulnerability_scan_inventory
+from wazuh_testing.db_interface import agent_db, cve_db
+from wazuh_testing import LOG_FILE_PATH
 from wazuh_testing.tools.monitoring import FileMonitor
-from wazuh_testing.tools import file
-
-# Marks
-pytestmark = [pytest.mark.server, pytest.mark.tier(level=0)]
-
-# variables
-test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-test_feed_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'data', 'feeds')
-configurations_path = os.path.join(test_data_path, 'wazuh_alert_vulnerability_removal.yaml')
-wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-wazuh_alert_monitor = FileMonitor(ALERT_FILE_PATH)
-test_packet_vendor = 'WazuhIntegrationTests'
-test_packet_version = '1.0.0'
-test_packet_version_not_vulnerable = '2.1.0'
-test_packet_0_name = 'wazuhintegrationpackage-0'
-test_packet_1_name = 'wazuhintegrationpackage-1'
-test_packet_0_cve = 'CVE-000'
-test_packet_1_cve = 'CVE-001'
-
-# Offline feeds
-rhel_oval_feed_path = os.path.join(test_feed_path, vd.CUSTOM_REDHAT_OVAL_FEED)
-rhel_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_REDHAT_JSON_FEED)
-nvd_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_NVD_FEED)
-parameters = [{
-    'RHEL_FEED_PATH': rhel_oval_feed_path,
-    'RHEL_JSON_FEED_PATH': rhel_json_feed_path,
-    'NVD_JSON_FEED_PATH': nvd_json_feed_path
-}]
-metadata = parameters
-
-# Configuration data
-configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
-
-# Read JSON data template
-nvd_vulnerabilities = file.read_json_file(nvd_json_feed_path)
-rhel_vulnerabilities = file.read_json_file(rhel_json_feed_path)
+from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.vulnerability_detector import event_monitor as evm
 
-# fixtures
-@pytest.fixture(scope='module', params=configurations)
-def get_configuration(request):
-    """Get configurations from the module."""
-    return request.param
 
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+CUSTOM_FEED_PATH = os.path.join(os.path.dirname(os.getcwd()), 'integration', 'test_vulnerability_detector', 'data')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'scan_result_alert_vulnerability_removal.yaml')
+test_alert_vuln_removal_path = os.path.join(TEST_CASES_PATH, 'cases_alert_vulnerability_removal.yaml')
```

Comment:
Rename to `cases_vulnerability_removal.yaml`

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2565/commits/50cd25f78e370246762e75c1eb6610a843281094